src/transformers/models/codegen/modeling_codegen.py

```diff
@@ -264,7 +264,6 @@ def forward(self, hidden_states: Optional[torch.FloatTensor]) -> torch.FloatTens
         return hidden_states
 
 
-# Copied from transformers.models.gptj.modeling_gptj.GPTJBlock with GPTJ->CodeGen
```

Comment:
removed copy line here.

Reply:
I think it is better to keep it, you can either propagate the tiny changes the made in `GPTJAttention` there or put `# Ignore Copy` on top of the init method